### PR TITLE
Feature/create layouts for regist login reset

### PR DESCRIPTION
**Pull request by Yojuks**
3 layouts - everyone have another route

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -4,6 +4,9 @@ import GlobalStyles from '@mui/material/GlobalStyles';
 import HomeScreen from 'screens/HomeScreen';
 import NavBar from 'components/Header/NavBar';
 import Footer from 'components/Footer/Footer';
+import LoginPage from 'components/LoginPage/LoginPage';
+import ForgotPassword from 'components/ForgotPassword/ForgotPassword';
+import Registration from './components/Registration/Registration';
```
Comment (steadyGuy) [inline frontend/src/App.tsx]: You called login as "LoginPage", but registration and forgot password without "page" prefix. Choose one variant how to call imported variables.

File: frontend/src/components/ForgotPassword/ForgotPassword.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  WrapButtonAndText
+} from './styles';
+
+function ForgotPassword() {
+  return (
+    <RegistrationFormWrapper>
+      <BorderForm>
+        <FormControl sx={{ width: '35ch' }}>
+          <WrapH1>
+            <h1>Forgot password?</h1>
```
Comment (steadyGuy) [inline frontend/src/components/ForgotPassword/ForgotPassword.tsx]: Typography with variant for "h1"

File: frontend/src/components/LoginPage/LoginPage.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  WrapH1
+} from 'components/ForgotPassword/styles';
+
+function LoginPage() {
+  return (
+    <RegistrationFormWrapper>
+      <BorderForm>
+        <FormControl sx={{ width: '35ch' }}>
+          <WrapH1>
+            <h1>Login</h1>
```
Comment (steadyGuy) [inline frontend/src/components/LoginPage/LoginPage.tsx]: Typography component for h1 with variant prop

**Review by steadyGuy — APPROVED** [review]
Some code are repeated, but it's not dramatically

File: frontend/src/App.tsx (hunk 1 above)
Comment (Yojuks) [reply frontend/src/App.tsx]: done

File: frontend/src/components/ForgotPassword/ForgotPassword.tsx (hunk 2 above)
Comment (Yojuks) [reply frontend/src/components/ForgotPassword/ForgotPassword.tsx]: done

File: frontend/src/components/LoginPage/LoginPage.tsx (hunk 3 above)
Comment (Yojuks) [reply frontend/src/components/LoginPage/LoginPage.tsx]: done

File: frontend/src/App.tsx (hunk 1 above)
Comment (6mahdapihka9) [reply frontend/src/App.tsx]: > You called login as "LoginPage", but registration and forgot password without "page" prefix. Choose one variant how to call imported variables.

It's called postfix :)

File: frontend/src/components/ForgotPassword/ForgotPassword.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+function ForgotPassword() {
+  return (
+    <RegistrationFormWrapper>
+      <BorderForm>
+        <FormControl sx={{ width: '35ch' }}>
+          <WrapH1>
+            <Typography>Forgot password?</Typography>
+          </WrapH1>
+
+          <TextFieldWrap>
```
Comment (6mahdapihka9) [inline frontend/src/components/ForgotPassword/ForgotPassword.tsx]: your own styled textfield blocks label in original textfield

therefore it's not displayed

File: frontend/src/components/ForgotPassword/ForgotPassword.tsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+            <TextField
+              placeholder="Please enter your email"
```
Comment (6mahdapihka9) [inline frontend/src/components/ForgotPassword/ForgotPassword.tsx:23]: placeholder doesn't work in MUI

instead use prop label and FormHelperText comp to show warnings or errors

**Review by ghost — APPROVED** [review]

File: frontend/src/components/ForgotPassword/styles.ts [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+export const StyledSpanOr = styled(Box)`
+  display: block;
+`;
+
+export const StyledSpanEnd = styled(StyledSpan)`
+  margin-top: 15px;
+`;
+
+export const TextFieldWrap = styled(TextField)`
```
Comment (6mahdapihka9) [inline frontend/src/components/ForgotPassword/styles.ts]: this blocks label to appear in all 3 forms. TextField comp is an input not a simple container

now its:
textfield
____textfield 
textfield

should be:
div
____textfield
div

File: frontend/src/components/Login/Login.tsx [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+function Login() {
+  return (
+    <RegistrationFormWrapper>
+      <BorderForm>
+        <FormControl sx={{ width: '35ch' }}>
+          <WrapH1>
+            <Typography>Login</Typography>
+          </WrapH1>
+          <TextFieldWrap>
```
Comment (6mahdapihka9) [inline frontend/src/components/Login/Login.tsx]: same situation here

**Review by 6mahdapihka9 — CHANGES_REQUESTED** [review]

File: frontend/src/components/Registration/Registration.tsx [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+              label="Password"
+              type="password"
+              autoComplete="current-password"
+            />
+          </TextFieldWrap>
+
+          <WrapButtonAndText>
+            <Button variant="contained">Create</Button>
+            <StyledSpan>
+              <span>Return to login</span>
```
Comment (6mahdapihka9) [inline frontend/src/components/Registration/Registration.tsx:42]: what does it mean?

should it lead to login from?
then why it's not a link?

File: frontend/src/components/ForgotPassword/ForgotPassword.tsx (hunk 5 above)
Comment (6mahdapihka9) [reply frontend/src/components/ForgotPassword/ForgotPassword.tsx:23]: you can also use prop fullWidth to stretch to make textfield a full size

**Review by 6mahdapihka9 — CHANGES_REQUESTED** [review]

File: frontend/src/components/Login/Login.tsx [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+            <TextField
+              placeholder="Please enter your email"
+              label="Password"
+              type="password"
+              autoComplete="current-password"
+            />
+          </TextFieldWrap>
+
+          <StyledSpanEnd>
+            <span>Forgot the password?</span>
```
Comment (6mahdapihka9) [inline frontend/src/components/Login/Login.tsx:40]: also should be a link to forgotPassword page

File: frontend/src/components/Login/Login.tsx [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+
+          <WrapButtonAndText>
+            <Button variant="contained">Login</Button>
+            <StyledSpan>
+              <span>or</span>
+            </StyledSpan>
+            <Button variant="contained">Sing in with google</Button>
+
+            <StyledSpan>
+              <span>don`t have an account</span>
```
Comment (6mahdapihka9) [inline frontend/src/components/Login/Login.tsx:51]: and this, link to registration

**Review by 6mahdapihka9 — CHANGES_REQUESTED** [review]

File: frontend/src/components/ForgotPassword/ForgotPassword.tsx (hunk 4 above)
Comment (Yojuks) [reply frontend/src/components/ForgotPassword/ForgotPassword.tsx]: done

File: frontend/src/components/ForgotPassword/styles.ts (hunk 6 above)
Comment (Yojuks) [reply frontend/src/components/ForgotPassword/styles.ts]: done

File: frontend/src/components/Login/Login.tsx (hunk 7 above)
Comment (Yojuks) [reply frontend/src/components/Login/Login.tsx]: done

File: frontend/src/components/Login/Login.tsx (hunk 9 above)
Comment (Yojuks) [reply frontend/src/components/Login/Login.tsx:40]: done

File: frontend/src/components/ForgotPassword/ForgotPassword.tsx (hunk 5 above)
Comment (Yojuks) [reply frontend/src/components/ForgotPassword/ForgotPassword.tsx:23]: done

**Review by 6mahdapihka9 — APPROVED** [review]

**Pull request merged**